Contributing.md

```diff
@@ -70,6 +70,12 @@ First, install [`mdbook`][mdbook] and [`nox`][nox]. Then, run
 nox -s build-guide -- --open
 ```
 
+To check all links in the guide are valid, also install [`lychee`][lychee] use the `check-guide` session instead:
```

Comment:
this sentence does not sound quite right

Reply:
Good point, I've added "and" - "and use the..."